sphinx_rtd_theme/static/js/theme.js

```diff
@@ -16,17 +16,22 @@ function ThemeNav () {
         isRunning: false
     };
 
-    nav.enable = function () {
+    nav.enable = function (withStickyNav) {
```

Comment:
What is the purpose of this addition? If configurable, this needs more information or documentation.

Reply:
This comes from #519. `enable()` was only called to enable sticky nav, while it was needed to decorate the navbar with the correct toggle options.
The extra bool was added so that a single entry point could be used to start the navigation logic (with or without sticky nav).
Documenting (with JSDoc?) of this file is a good idea, perhaps something for a separate issue?

Reply:
Yeah, not important here I think, but having our public API documented would make sense for folks extending or repurposing the theme.